src/DefaultBackedSettings.ts

```diff
@@ -0,0 +1,36 @@
+import { fileBackedObject } from "./FileBackedObject";
+
+import fs = require("fs");
+
+export function defaultBackedObject<T>(path: string, overwritePath: string): T {
+    const defaults = fs.readFileSync(path, "utf-8");
+    const overwrite = fs.readFileSync(overwritePath, "utf-8");
+    const obj = Object.assign({}, JSON.parse(defaults), JSON.parse(overwrite));
+
+    const proxy = {
+        set(object: any, property: string, value: any, receiver: any) {
+            Reflect.set(object, property, value, receiver);
+            try {
+                fs.writeFileSync(path, JSON.stringify(obj));
```

Comment:
Any changes made should probably go to the overwrite path rather than the regular one

Reply:
I'm thinking the overwrite path is the changes you always want the same?  
Also, setting a value is most likely never getting used for this, as settings are read-only at the moment